iceoryx_examples/iceperf/base.cpp

```diff
@@ -16,54 +17,55 @@
 #include "base.hpp"
 
 
-void IcePerfBase::prePingPongLeader(uint32_t payloadSizeInBytes) noexcept
+void IcePerfBase::preLatencyPerfTestLeader(uint32_t payloadSizeInBytes) noexcept
 {
-    sendPerfTopic(payloadSizeInBytes, true);
+    sendPerfTopic(payloadSizeInBytes, RunFlag::RUN);
 }
 
-void IcePerfBase::postPingPongLeader() noexcept
+void IcePerfBase::postLatencyPerfTestLeader() noexcept
 {
     // Wait for the last response
     receivePerfTopic();
 }
 
 void IcePerfBase::releaseFollower() noexcept
 {
-    sendPerfTopic(sizeof(PerfTopic), false);
+    sendPerfTopic(sizeof(PerfTopic), RunFlag::STOP);
 }
 
-double IcePerfBase::pingPongLeader(uint64_t numRoundTrips) noexcept
+double IcePerfBase::latencyPerfTestLeader(uint64_t numRoundTrips) noexcept
 {
     auto start = std::chrono::high_resolution_clock::now();
 
     // run the performance test
     for (auto i = 0U; i < numRoundTrips; ++i)
     {
         auto perfTopic = receivePerfTopic();
-        sendPerfTopic(perfTopic.payloadSize, true);
+        sendPerfTopic(perfTopic.payloadSize, RunFlag::RUN);
```

Comment:
Could you please rename the enum from `RunFlag` to `RunForest`.
It would read so nicely: `RunForest::RUN`

:)